### PR TITLE
Deprecate eGRIDRegionCode

### PR DESCRIPTION
**Pull request by JieXiong9119**
Follow-up updates on #358

### CODE REVIEW

**Comment by JieXiong9119** [comment]
Need to update v2-to-v3 translator to include this change

**Review by macintoshpie — COMMENTED** [review]
Schema changes looks perf, let me know if you want a second pair of eyes on the translator stuff

**Comment by JieXiong9119** [comment]
> Schema changes looks perf, let me know if you want a second pair of eyes on the translator stuff

Should be good for final review now.

**Review by nllong — APPROVED** [review]
Looks great!

File: translation/v2_to_v3.xsl [hunk 1]
```diff
@@ -148,17 +148,25 @@ Version: 0.1.0
...
     </xsl:element>
   </xsl:template>
-  
   <!-- Transform auc:WindowID element under auc:Side to auc:WindowIDs/auc:WindowID element -->
   <xsl:template match="auc:WindowID">
     <xsl:element name="auc:WindowIDs">
       <xsl:copy-of select="."/>
     </xsl:element>
   </xsl:template>
+  <!-- Transform auc:WindowID element under auc:Side to auc:WindowIDs/auc:WindowID element -->
```
Comment (macintoshpie) [inline translation/v2_to_v3.xsl:159]:
```suggestion
  <!-- Transform auc:DoorID element under auc:Side to auc:DoorIDs/auc:DoorID element -->
```

**Review by macintoshpie — COMMENTED** [review]
Looks good to me, we just need to update a comment in the translator

File: translation/v2_to_v3.xsl (hunk 1 above)
Comment (JieXiong9119) [reply translation/v2_to_v3.xsl]: That's a stupid mistake..

**Pull request merged**